scripts/buildhost/README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.com/algorand/go-algorand.svg?token=25XP72ADqbCQJ3TJVC9S&branch=master)](https://travis-ci.com/algorand/go-algorand)
+[![Build Status](https://travis-ci.com/algorand/go-algorand.svg?branch=master)](https://travis-ci.com/algorand/go-algorand)
```

Comment:
this line is not needed. The build status is irrelevant on this file.
( we don't compile the buildhost directory, so there is no "build status" here )

Reply:
That is a very good point. Fixed that and a formatting bug.